test-app/src/widgets/AlertDialog.js

```diff
@@ -2,9 +2,9 @@ import AccessibilityModule from '@curriculumassociates/createjs-accessibility';
 import Button from './Button';
 
 export default class AlertDialog extends createjs.Container {
-  constructor({ tabIndex, doneCallback, cancelCallback }) {
+  constructor({ buttonTabIndex, doneCallback, cancelCallback }) {
```

Comment:
Renamed for clarity that this is not the tabIndex of the alertdialog as a whole, but is the tabIndex of the buttons in the alertdialog